docs/embedding/sdk/introduction.md

```diff
@@ -18,7 +18,7 @@ Here's the [Shoppy source code](https://github.com/metabase/shoppy).
 
 ## Embedded analytics SDK prerequisites
 
-- React application using React 17 or React 18.
+- React application using React 18 or React 19.
```

Comment:
In the backported PR, I will modify it to `17, 18, or 19`. Metabase 54 is not compatible with React 17 at the moment because of Mantine v7.